pages/index.js

```diff
@@ -116,20 +116,21 @@ function HomePage() {
         </div>
       </Container>
       
-      <Container className="text-center py-12" wide>
+      <Container className="text-center py-6" wide>
         <h3 className="text-primary-800 text-3xl font-bold md:text-4xl mb-4">Platinum Sponsors</h3>
-        <p className="mt-2 text-base leading-6 text-gray-500 md:w-1/2 md:mx-auto">
-          These great organizations are already supporting AsyncAPI. Want to become a sponsor? <a href="https://opencollective.com/asyncapi" target="_blank" className="text-primary-600 font-medium" rel="noreferrer">Support us!</a>
-        </p>
-        <Sponsors className="mt-8" showSupportBanner={false} />
+        <Sponsors className="mt-4" showSupportBanner={false} />
       </Container>
 
-      <Container className="text-center py-12" wide>
+      <Container className="text-center py-6" wide>
         <h3 className="text-primary-800 text-3xl font-bold md:text-4xl mb-4">Gold Sponsors</h3>
+        <GoldSponsors className="mt-4" showSupportBanner={false} />
+      </Container>
+      
+      <Container className="text-center py-6" wide>
+        <h3 className="text-primary-800 text-2xl font-bold md:text-3xl mb-4">Want to Sponsor Us?</h3>
```

Comment:
I think here we can just remove `md:text-4xl` as I think the `text-2xl` size is perfect to not compete with the other titles above. I think also we can change the font weight from `font-bold` to `font-semibold`. What do you think?

Reply:
@mcturco When I had removed `md:text-4xl` from classes, I saw that the heading `Want to Sponsor Us` is getting way too smaller especially on big screens. So, I have made it `md:text-3xl` to make it a bit small from the Sponsor heading. And regarding `font-semibold`, that doesn't make any difference in the preview. You can surely check this out using Developer Tools.

Reply:
Oops sorry, I think I meant the `md:text-3xl` only removing this class from the `Want to sponsor us?` and then changing the font weight to semibold. Here is a screenshot of me making these changes in the preview:

![Screen Shot 2022-01-14 at 2 55 48 PM](https://user-images.githubusercontent.com/60163079/149577055-1723405a-e0ad-4b1c-8269-355232b6c7d8.png)

The semibold works for me in my preview, so that could just be an issue with your `system-ui` font, perhaps it doesn't have this weight. We are changing the fonts very soon and there will be a semibold weight, so maybe just add that class there so its ready for when we merge the new styles.

Reply:
Ok Fine. I will make changes to this.

Reply:
@mcturco Do check now if everything is according to the changes you want to make.